types/tabulator-tables/index.d.ts

```diff
@@ -764,7 +764,7 @@ declare namespace Tabulator {
     interface ColumnDefinition extends ColumnLayout, CellCallbacks {
         // Layout
         /** sets the text alignment for this column */
-        align?: "left" | "center" | "right"; // Align?
+        align?: "left" | "center" | "right" | string; // Align?
```

Comment:
Documentation only states left,center,right.
http://tabulator.info/docs/4.2/columns#definition

Reply:
I undestand what you say but this row is uncorrect. 
I will do another commit only for this. 
Thanks